cli/templates/project/src/pages/_app.jsx

```diff
@@ -1,17 +1,13 @@
-import { useRouter } from "next/router"
+import { AppContextProviders, Page } from '@local/components/app'
 
-import { AppContextProviders, Page } from "@local/components/app"
-
-import "@local/styles/globals.scss"
+import '@local/styles/globals.scss'
 
 export default function App({ Component, pageProps }) {
```

Comment:
Hey @andrewrubin how do you feel about changing `pageProps` to something else? Or is that just too nit picky?

Reply:
I thought pageProps was a Next JS naming convention, but maybe I'm mistaken! I think it makes sense as is, but I'm curious what you would suggest we change it to

Reply:
I think you are right. I initially thought it was because they were props for the page context. All good!